client/src/screens/Client/Home/components/modal/NewOrEditEvent.js

```diff
@@ -308,12 +307,13 @@ const NewOrEditEvent = ({
                 label="Start"
                 value={calEvent.start_dt}
                 placeholder="2020/10/10 10:00"
-                onChange={(date) =>
+                onChange={(date) => {
+                  let property = "start_dt";
```

Comment:
I am curious to know why is it necessary to assign `start_dt` into a variable?

Reply:
I did it to fix the eslint warning.